app/components/UploadTrack/index.js

```diff
@@ -0,0 +1,202 @@
+import React, { memo, useState } from 'react';
+import { Steps, Button, Form, Input, DatePicker, Upload } from 'antd';
+import { UploadOutlined } from '@ant-design/icons';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import * as colors from '@app/themes/colors';
+import { TrackDetailsComponent } from '../TrackDetailsComponent/index';
+
+const StyledCard = styled.div`
+  && {
+    background-color: ${colors.background};
+    padding: 5rem;
+  }
+`;
+
+const StyledInput = styled(Input)`
+  && {
+    width: 25rem;
+  }
+`;
+
+const StyledFormItem = styled(Form.Item)`
+  && {
+    margin: 2rem;
+    color: ${colors.whiteText};
+    wrapper
+  }
+  .ant-form-item-required {
+    color: ${colors.whiteText};
+    font-size: 1rem;
+  }
+`;
+
+const TrackContainer = styled.div`
+  && {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: ${colors.background};
+    min-width: 100vw;
+    border: 1px solid ${colors.primary};
+    min-height: 100vh;
+  }
+`;
+
+const { Step } = Steps;
+
+export function UploadTrack() {
+  const [data, setData] = useState({});
+  const [current, setCurrent] = useState(0);
+  const onFinish = (values) => {
+    setData(Object.assign(data, values));
+    setCurrent(current + 1);
+  };
+
+  const prev = () => {
+    setCurrent(current - 1);
+  };
+
+  const contentOne = () => {
```

Comment:
all functions that are responsible for "rendering" should be prefixed with `render` 

Let's please also rename to depict what it is rendering rather than "contentOne"